release-doc-in-process/all_vf.TTL

```diff
@@ -46,6 +46,7 @@ vf:RecipeProcess a          owl:Class ;
 
 vf:RecipeFlow a             owl:Class ;
         rdfs:label          "vf:RecipeFlow" ;
+        rdfs:subClassOf     time:TemporalEntity ;
```

Comment:
I still am not comfortable with the subclassing - the broad understanding of that term is like "is a", "is a type of", etc. And I think it is conceptually misleading.

Reply:
It has very precise definition explaining exactly what it causes: https://www.w3.org/TR/rdf-schema/#ch_subclassof

I removed it for now since no need to argue about it in this PR